### PR TITLE
fix: placeholder url for missing pfps

### PR DESCRIPTION
**Pull request by colinwilliams91**
mailtos and placeholder images, hack for logo color theme...
NEEDS BETTER SOLUTION

### CODE REVIEW

File: app/organizations.ts [hunk 1]
```diff
@@ -99,11 +96,10 @@ export const organizationsStore: Organizations = {
...
     "organizers": [
       {
-        "name":"Some Guy McPlaceholder",
-        "pfp": "/organizersPfp/ben.webp",
+        "name":"Calder",
+        "pfp": "https://via.placeholder.com/158x158",
         "links": [
-          {"linkedin": "https://www.linkedin.com/in/benjamin-eckel-b025831a3/"},
-          {"email": "benjamin.eckel@gmail.com"}
+          {"website": "https://nolagamedev.wordpress.com/contact/"}
```
Comment (Aeshus) [inline app/organizations.ts:102]: I think that the URL should be [http://neworleansgamedev.com](http://neworleansgamedev.com) (note, the URL is http-only for some reason...)

They use this url on their Discord and all of their official communication channels so I'm assuming that the wordpress site was just an old site that they used to use.

Comment (Aeshus) [reply app/organizations.ts:102]: They have an organization-wide email (found on their Discord) if that would be better to put there.
`org@neworleansgamedev.com`

Comment (colinwilliams91) [reply app/organizations.ts:102]: Appreciate your help @Aeshus 

I implemented your suggestions and am about to push up. I elected to use the `contact` email address. I reached out to Calder and he gave me a date he would provide me assets and contact info, but that date came and went so I don't want to pester him about it. I will reach out again soon.

**Review by Aeshus — COMMENTED** [review]
Looks good to me, though I don't have any experience with NextJS or NextUI apps (so I can't really review much of the typescript stuff).

It would probably be good to get @codingMustache to take a look at it too.

File: app/layout.tsx [hunk 2]
```diff
@@ -86,7 +88,7 @@ export default function RootLayout({
   }, []);
 
   return (
-    <html lang="en">
+    <html lang="en" id="html">
```
Comment (Aeshus) [inline app/layout.tsx:91]: What does `id="html"` do? 
It doesn't seem to have any styling attached and doesn't seem to make a difference if it's there or not?

**Review by codingMustache — APPROVED** [review]

**Pull request merged**